dockers/Screenshotter/screenshotter.js

```diff
@@ -358,16 +377,42 @@ function findHostIP() {
 //////////////////////////////////////////////////////////////////////
 // Take the screenshots
 
-let countdown = listOfCases.length;
+let countdown = listOfCases.length * modes.length;
 
 let exitStatus = 0;
 const listOfFailed = [];
 
+const doctypes = {
+    "no-quirks": "<!DOCTYPE html>",
+    "limited-quirks": '<!DOCTYPE HTML PUBLIC ' +
+        '"-//W3C//DTD HTML 4.01 Transitional//EN" ' +
+        '"http://www.w3.org/TR/html4/loose.dtd">',
+    "quirks": '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Draft//">',
+};
+
+// Use two characters per abbreviation for better alignment of output
+const qabbr = {
+    "no-quirks": "nq",
+    "limited-quirks": "lq",
+    "quirks": " q",
+};
+
 function takeScreenshots() {
-    listOfCases.forEach(takeScreenshot);
+    const html = fs.readFileSync(require.resolve(
+        "../../test/screenshotter/test.html"));
+    function handler(req, res, next) {
+        res.send(doctypes[req.query.mode] + "\n" + html);
+    }
+    app.get("/ss-render.html", handler);
+    app.get("/babel/ss-render.html", handler);
```

Comment:
I tried removing the second `get` but the screenshotter hung after starting the chrome tests after completing the firefox tests.  Not sure why since server.js doesn't serve this URL.